mapper/src/main/java/com/ibm/mapper/model/algorithms/PBKDF1.java

```diff
@@ -46,10 +46,14 @@ public String asString() {
             sb.append(digest.get().asString());
             return sb.toString();
         }
-
         return this.name;
     }
 
+    @Override
+    public @NotNull String getName() {
+        return asString();
+    }
+
```

Comment:
I'm not sure why you would have to redefine `getName` and cannot only rely on `asString` for this case (and for `BPKDF2`)?